### PR TITLE
Changed topic names for payments

### PR DESCRIPTION
**Pull request by marsoy**

### CODE REVIEW

File: crema/choices.py [hunk 1]
```diff
@@ -17,6 +17,8 @@ class EventType(Enum):
     ASSISTANCE_REMINDER = "ASSISTANCE_REMINDER"
     ASSISTANCE_ACTIVITY = "ASSISTANCE_ACTIVITY"
     WHATSAPP_COMM = "WHATSAPP_COMM"
+    COMMENT = "COMMENT"
```
Comment (rohitladdha) [inline crema/choices.py:20]: Is this related to Payments?

Comment (rohitladdha) [reply crema/choices.py:20]: @akshaywattamwar01 Can we rename it to `OPS_COMMENT`? `COMMENT` seemed too generic to me.

Comment (akshaywattamwar01) [reply crema/choices.py:20]: sure.
